lib/report/k8s.go

```diff
@@ -58,25 +58,33 @@ func KubernetesInfo(ctx context.Context, runner utils.CommandRunner) Collectors
 				utils.PlanetCommand(kubectl.Command("describe", resourceType, "--namespace", namespace))...))
 		}
 
+		logger := log.WithField("namespace", namespace)
 		// fetch pod logs
 		pods, err := kubectl.GetPods(ctx, namespace, runner)
 		if err != nil {
-			log.Errorf("failed to query pods in namespace %v: %v", namespace, trace.DebugReport(err))
+			logger.WithError(err).Warn("Failed to query pods.")
 			continue
 		}
 		for _, pod := range pods {
 			containers, err := kubectl.GetPodContainers(ctx, namespace, pod, runner)
 			if err != nil {
-				log.Errorf("failed to query container in pod %v in namespace %v: %v",
-					pod, namespace, trace.DebugReport(err))
+				logger.WithFields(log.Fields{
+					log.ErrorKey: err,
+					"pod":        pod,
+				}).Warn("Failed to query container.")
 				continue
 			}
 			for _, container := range containers {
 				name := fmt.Sprintf("k8s-logs-%v-%v-%v", namespace, pod, container)
 				commands = append(commands, Cmd(name, utils.PlanetCommand(kubectl.Command("logs", pod,
 					"--namespace", namespace,
-					fmt.Sprintf("-c=%v", container)))...),
-				)
+					fmt.Sprintf("-c=%v", container)))...))
```

Comment:
Doesn't cluster-info dump also collect container logs?

Reply:
Yes it does and hence I went forward and deleted all ad-hoc captures for kubernetes resources leaving only the `kubectl cluster-info dump` and the captures of previous containers since kubectl is not capturing them.